cycledash/genotypes.py

```diff
@@ -92,16 +92,16 @@ def get(vcf_id, query, with_stats=True):
      limit: 250
     }
     """
-    query = _annotate_query_with_types(query, spec(vcf_id))
-    compare_to_vcf_id = query.get('compareToVcfId')
+    query = _annotate_query_with_types(query, spec(run_id))
+    compare_to_run_id = query.get('compareToVcfId')
```

Comment:
Yeah…this is part of a slow migration from VCF to run. I've just been changing it where I touch the code. I'll change `compareToVcfId` to `compareToRunIf`.